hub/package-manager/winget/upgrade.md

```diff
@@ -75,11 +76,11 @@ The following example shows upgrading all apps
 winget upgrade --all
 ```
 
-## Using list and upgrade
+## Using **upgrade**
 
-It is common to use the [**list**](.\list.md) command to identify apps in need of an update, and then to use **upgrade** to install the latest.
+To identify which apps are in need of an update, simply use **upgrade** without any arguments to show all available upgrades. 
```

Comment:
Is list no longer supported?

Reply:
List is still supported. I added back the description and example for how to use the list command with upgrade as I feel that it is still useful and relevant for identifying an update for a specific package.